### PR TITLE
Train utils

### PR DESCRIPTION
**Pull request by corochann**
Introduced `run_train` util method. It can be used for typical training.

### CODE REVIEW

File: chainer_chemistry/utils/train_utils.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        device (Device): chainer Device
+        converter (callable):
+        use_default_extensions (bool): If `True`, default extensions are added
+            to `trainer`.
+        resume_path (None or str): If specified, `trainer` is resumed with this
+            serialized file.
+    """
+    if optimizer is None:
+        # Use Adam optimizer as default
+        optimizer = optimizers.Adam()
```
Comment (mottodora) [inline chainer_chemistry/utils/train_utils.py]: `elif isinstance(optimizer, Optimizer):` ?

**Comment by corochann** [comment]
Please merge if it is ok

**Pull request merged**